ci_group/revolve2/ci_group/drones_v1.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+from revolve2.modular_robot.body.drone_v1 import DroneCoreV1, MotorV1, DroneBodyV1
+from pyrr import Quaternion, Vector3
+
+
+def all() -> list[DroneBodyV1]:
+    """
+    Get a list of all standard module robots.
+
+    :returns: The list of robots.
+    """
+    return [cross_v1()]
+
+def cross_v1() -> DroneBodyV1:
```

Comment:
v1 is only in the names since we have v1 and v2 bodies, im not sure if you plan to do iterations on your drone system. If not i would leave out the v part

Reply:
Yeah, this was an internal debate I had. In the future, I imagine I may create different versions with different part specs. But I will also just keep developing this one to make it more realistic. I think for now I might take away the versioning to save confusion.